venues/MIDL.io/2019/Conference/python/init_conference.py

```diff
@@ -34,7 +34,7 @@
         },
         "remove if rejected": {
             "order": 12,
-            "value-checkbox": "Remove submission if paper is rejected.",
+            "value-checkbox": "(optional) Remove submission if paper is rejected.",
```

Comment:
the ui shoes the required field, is this still needed?

Reply:
This was a specific request resulting from the demo last Friday.